utils/build.ps1

```diff
@@ -1237,11 +1237,23 @@ function Build-SPMProject {
     [string[]] $AdditionalArguments
   )
 
+  switch ($Action) {
+    Build {
+      $ActionForOutput = "Building"
+    }
+    Test {
+      $ActionForOutput = "Testing"
+    }
+    TestParallel {
+      $ActionForOutput = "Testing"
+    }
+  }
```

Comment:
I wonder if we can rewrite this as:

```suggestion
$ActionForOutput = switch ($Action) {
Build { "Building" }
Test { "Testing" }
TestParallel { "Testing" }
}
```

Reply:
Ah, nice. Didn’t know that was possible.